grcs/0020-knowledge-graph.md

```diff
@@ -0,0 +1,527 @@
+---
+GRC: 20
+Title: Knowledge Graph
+Authors: Yaniv Tal, Byron Guina
+Created: 2024-11-21
+Stage: Proposal
```

Comment:
Ideally this is just "Draft" as the single state we're keeping for GRCs (in GIP-0061: https://github.com/graphprotocol/graph-improvement-proposals/pull/30/files we're working to update the process to simplify it)